### PR TITLE
Added machine learning resources in python section

### PR DESCRIPTION
**Pull request by ChinmayBH**
The contributed repository is a curated list of awesome machine learning frameworks, libraries and software. Further more there are lots of relevant books, courses and events are added in this repository.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -2095,6 +2095,20 @@
...
+            "technologies": [
+                "Python"
+            ],
+            "description": "A curated list of awesome machine learning frameworks, libraries and software."
         }
+
+
+
+
+
```
Comment (StackOverflowIsBetterThanAnyAI) [inline data.json:2112]: Please remove these empty lines.

**Review by jakobgamer16 — APPROVED** [review]

**Review by jakobgamer16 — APPROVED** [review]

File: data.json (hunk 1 above)
Comment (ChinmayBH) [reply data.json]: I resolved this issue, please check, thanks!

**Review by sammyhori — COMMENTED** [review]
None of the repositories issues use labels

**Comment by sammyhori** [comment]
As this pull request goes against the contributing guideline of:

> Ensure the contributed repository is actively maintained, has a supportive community, and _issues with appropriate labels_.

I am closing this pull request. Feel free to reopen this PR if some issues with the correct label are made.

**Pull request closed without merging**